rust/windows-client/src-tauri/src/client/gui.rs

```diff
@@ -220,13 +220,14 @@ fn handle_system_tray_event(app: &tauri::AppHandle, event: TrayMenuEvent) -> Res
         }
         TrayMenuEvent::SignIn => app
             .try_state::<Managed>()
-            .ok_or_else(|| anyhow!("getting ctlr_tx state"))?
+            .ok_or_else(|| anyhow!("couldn't get ctlr_tx state"))?
             .ctlr_tx
             .blocking_send(ControllerRequest::SignIn)?,
-        TrayMenuEvent::SignOut => {
-            keyring_entry()?.delete_password()?;
-            app.tray_handle().set_menu(signed_out_menu())?;
-        }
+        TrayMenuEvent::SignOut => app
+            .try_state::<Managed>()
+            .ok_or_else(|| anyhow!("couldn't get ctlr_tx state"))?
+            .ctlr_tx
+            .blocking_send(ControllerRequest::SignOut)?,
```

Comment:
Is there a point separating the `handle_system_tray_event` and `run_controller` tasks? seems like mostly we want to send these event to the other task.

Reply:
Yeah, we should just send the event to the Controller directly. I'll put that down for after some other PRs land that affect the controller.